libs/datamodel/parser-database/src/walkers/index.rs

```diff
@@ -83,14 +99,17 @@ impl<'ast, 'db> IndexWalker<'ast, 'db> {
         self.index_attribute.source_field.is_some()
     }
 
+    /// Is this an `@(@)unique`?
     pub fn is_unique(self) -> bool {
         self.index_attribute.is_unique()
     }
 
+    /// Is this an `@(@)fulltext`?
```

Comment:
`a @@fulltext`

Reply:
Hehe I had thoughts on that. Since I pronounce that `at at fulltext`, "a" feels unnatural. I'll ask the docs team what we should do.

Reply:
I don't think there is `@fulltext`, so the `()` can go.